src/main/java/com/eternalcode/lobbyheads/head/hologram/HologramService.java

```diff
@@ -55,17 +61,11 @@ public void createHologram(OfflinePlayer player, Position position, String headN
     public void loadHolograms() {
         String defaultHeadFormat = this.config.headSection.defaultHeadFormat;
 
-        for (Head head : this.config.heads) {
+        for (Head head : this.headManager.getHeads()) {
```

Comment:
I don't understand where is performance improvement. Method `.getHeads()` will copy list of heads via a stream that is no better than reference to list.